consent-ws/src/main/java/org/broadinstitute/consent/http/service/UseRestrictionConverter.java

```diff
@@ -88,6 +88,20 @@ else if (selectedGender.equalsIgnoreCase("F"))
         if (pediatricsOnly) {
             dataUseDTO.setPediatric(true);
         }
+
+
+        if(Boolean.valueOf(form.getOrDefault("poa", false).toString())){
```

Comment:
Nit spacing throughout. Please check your IDE settings to ensure that the space settings are consistent with the rest of the code. For example, this line should be:
```
if (Boolean.valueOf(form.getOrDefault("poa", false).toString())) {
```

Reply:
Sorry for that, this is fixed now